xml-csharp/XMLDeserializationInCsharp/XMLDeserializationInCsharp/Person.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace XMLDeserializationInCsharp
+{
+    public class Person
+    {
+        public string Name { get; set; }
+        public int Age { get; set; }
+    }
```

Comment:
This would be a good use case example to show deserializing of records. I would recommend that we change this to a record and then we will be able to highlight how deserialization works with records. The big catch is that we need to define a parameterless constructor on the object, but we can define it privately.
`public record Person(string Name, int Age)
{
    private Person() : this("", int.MinValue) {}
}`